maquetado/scss/_styles.scss

```diff
@@ -0,0 +1,13 @@
+// Imports
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+// Common
+body {
+  font-family: $font-main;
+}
+
+// Sections
+.section {
+  margin: 70px 100px 100px;
```

Comment:
Mmm, think of a screen that's big enough to make this margins really small. We want the content to be centered so you can do better by setting a max-width to the content and margin auto in the sides

Reply:
Fixed   ;)